templates/README_imgs.md

```diff
@@ -0,0 +1,9 @@
+# [TITLE GOES HERE]
```

Comment:
Can we make this a `README_imgs.tpl` or something along those lines just to make it a bit more obvious?